lua/attempt/config.lua

```diff
@@ -3,41 +3,44 @@ local M = {}
 local unix = vim.fn.has 'unix' == 1
 local path_separator = unix and '/' or '\\'
 
+-- local cpp_compiler = "g++"
+-- local c_compiler = "gcc"
```

Comment:
I'd remove these comments, unless you want to submit both changes in this PR (which is fine by me)